.lintrunner.toml

```diff
@@ -259,7 +259,6 @@ exclude_patterns = [
     '**/fb/**',
     '**/*pb.h',
     '**/*CUDA*',
-    '**/cuda/*pp',
```

Comment:
Does this actually run? Is the CUDA lintrunner running yet?

Reply:
We switch to CUDA docker first then the exclusion patterns will be dropped gradually.